src/ngx_http_lua_kong_module.c

```diff
@@ -48,11 +49,23 @@ static ngx_http_module_t ngx_http_lua_kong_module_ctx = {
     NULL                                     /* merge location configuration */
 };
 
+static ngx_command_t ngx_http_lua_kong_cmds[] = {
```

Comment:
```suggestion
static ngx_command_t ngx_http_lua_kong_commands[] = {
```

This is consistent with Nginx's module naming conventions.